### PR TITLE
subliminal 0.7.5

### PR DESCRIPTION
**Pull request by tdsmith**

### CODE REVIEW

File: Library/Formula/subliminal.rb [hunk 1]
```diff
@@ -15,93 +13,106 @@ class Subliminal < Formula
 
   depends_on :python if MacOS.version <= :snow_leopard
 
-  resource "charade" do
-    url "https://pypi.python.org/packages/source/c/charade/charade-1.0.3.tar.gz"
-    sha1 "6bf65ae4b60d13e89bd7afe1704eef5bd0b787bc"
-  end
-
-  # Don't upgrade >0.7.0 - Package incompatible with above.
-  resource "guessit" do
```
Comment (DomT4) [inline Library/Formula/subliminal.rb]: You bumped ` guessit `. Is upstream Subliminal compatible with newer versions of ` guessit ` finally or?

Comment (tdsmith) [inline Library/Formula/subliminal.rb]: I captured what pip installs, so: yep!

Comment (DomT4) [inline Library/Formula/subliminal.rb]: Nice, cool.

**Comment by mikemcquaid** [comment]
:+1:

File: Library/Formula/subliminal.rb [hunk 2]
```diff
@@ -15,93 +13,111 @@ class Subliminal < Formula
...
+    touch libexec/"vendor/lib/python2.7/site-packages/dogpile/__init__.py"
+
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    system "python", *Language::Python.setup_install_args(libexec)
+    bin.install Dir[libexec/"bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
   end
 
   test do
+    mkdir ".config"
```
Comment (mikemcquaid) [inline Library/Formula/subliminal.rb]: Might want to make this `#{ENV["HOME"]}/.config` just to clarify what's going on here?

**Pull request closed without merging**